gRPC-C++.podspec

```diff
@@ -23,15 +23,15 @@
 Pod::Spec.new do |s|
   s.name     = 'gRPC-C++'
   # TODO (mxyan): use version that match gRPC version when pod is stabilized
-  # version = '1.20.0-pre3'
-  version = '0.0.8-pre3'
+  # version = '1.20.0'
+  version = '0.0.8'
```

Comment:
@muxi  @srini100 I don't know what "0.0.8" signifies and why we change it (-pre3 to regular). But the actual version string is commented so is not even used. Does this need to be looked at?